speclang_modern.tex

```diff
@@ -2158,6 +2193,9 @@ \subsection{Measures and non-terminating functions}
 terminate, while the one on line 8 is verified (it belongs to the body of
 \lstinline{f} that might not terminate).
 
+Here, statement \lstinline|terminates| allows to verify that some code section
```

Comment:
The term *statement terminates* is a shortcut to ban.

Reply:
Done (not pushed)